### ISSUE TITLE
FloatWithUncertainties doc page - seems to have been copied from ComplexWithUncertainties and not changed

### ISSUE THREAD

**Issue by mikehagerty** [obspy/obspy#3101]
### Documentation Link

https://docs.obspy.org/packages/autogen/obspy.core.util.obspy_types.FloatWithUncertainties.html

### Problem

Doc page shows FloatWithUncertainties as having attributes:
  real
  imag

Seems like this page was probably copied from ComplexWithUncertainties and
not modified all the way

**Comment by megies** [comment]
I'm actually puzzled by this one, I don't see where the imag and real in the docs page comes from..

https://docs.obspy.org/_modules/obspy/core/util/obspy_types.html#FloatWithUncertainties

**Comment by megies** [comment]
No, I mean, I really have zero idea right now, how these things end up on the sphinx compiled docs.. I dont see anything about imag/real in the source code of FloatWithUncertainties.
It is inheriting from `CustomFloat` which inherits from `float` and there is no imag/real anywhere in any of those. I'm puzzled.

**Comment by ThomasLecocq** [comment]
aha, indeed!
